Samples/RustSqueezenet/copy_collateral.bat

```diff
@@ -0,0 +1,3 @@
+xcopy ..\..\SharedContent\models\SqueezeNet.onnx .\target\debug\ /Y
```

Comment:
robocopy instead of xcopy prevents unnecessary copies. Would be nice if it worked in release too, and if it was part of build.rs instead of a separate bat.

Reply:
Currently there isn't a good way to incorporate copying resource files during cargo build. You can see in issues:
- https://github.com/rust-lang/cargo/issues/5305
- https://github.com/rust-lang/cargo/issues/1759#issuecomment-391694959

Reply:
I'm asking winrt-rs folks, just to be sure.